cmd/kk/pkg/images/utils.go

```diff
@@ -173,11 +173,18 @@ func NewManifestSpec(image string, entries []manifesttypes.ManifestEntry) manife
 
 func validateImageName(imageFullName string) error {
 	image := strings.Split(imageFullName, "/")
-	if len(image) != 3 {
-		return errors.Errorf("image %s is invalid, only the format \"registry/namespace/name:tag\" is supported", imageFullName)
+	if len(image) >= 3 {
+		return errors.Errorf("image %s is invalid,image PATH need contain  at least two slash-separated", imageFullName)
```

Comment:
Extra whitespace between `contain  at` is not necessary. A whitespace after comma is necessary. `invalid,image`

Reply:
ok